tensorrt_llm/_torch/autotuner.py

```
@@ -1078,6 +1078,15 @@ def _profile_runners(
                             **kwargs,
                         )
                 except Exception as e:
+                    # Synchronize to clear any pending CUDA errors left by
+                    # the failed profiling kernel, preventing stale errors
+                    # from propagating to subsequent profiling iterations
+                    # or the inference forward pass.
+                    try:
+                        torch.cuda.synchronize()
+                    except Exception:
+                        pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)